lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -523,18 +523,7 @@ function dosomething_reportback_view_entity($entity) {
     $fid = array_pop($entity->fids);
   }
 
-
-  // @TODO Can we templatize/DRY some of this? It's fairly similiar code in multiple places.
-  $disable_reactions = dosomething_kudos_disable_reactions($entity->nid);
-
-  if (! $disable_reactions) {
-    $kudos = [];
-    $kudos['rbid'] = $entity->rbid;
-    $kudos['disable_reactions'] = $disable_reactions;
-    $kudos['allowed_reactions'] = [dosomething_kudos_get_term_id_by_name('heart')];
-    $kudos['reaction_totals'] = ['heart' => dosomething_kudos_get_total_for_file_by_term_name($entity->rbid, 'heart')];
-    $kudos['existing_kids'] = dosomething_kudos_get_kudos_for_user_by_file($entity->rbid);
-  }
```

Comment:
right but going back to what i wrote here: https://github.com/DoSomething/phoenix/pull/6689/files/e839e044531f01b9fe7635bbdf77a4b560469836#r69947073

we would then need to write this code 3 times. the point in self enclosing everything was to make it 1 line everywhere, no room for error. if you'd still prefer having the disable check in each file I will go back to that.